src/main/java/hudson/plugins/gradle/Gradle.java

```diff
@@ -185,35 +189,6 @@ public void setPassAllAsProjectProperties(boolean passAllAsProjectProperties) {
         this.passAllAsProjectProperties = passAllAsProjectProperties;
     }
 
-    public GradleInstallation getGradle() {
```

Comment:
Removing this here won't break any API compatibility, will it?

Reply:
This was only used by the config.jelly with was also adjusted to now use getGradleName insted, so this should not be an problem.

Reply:
Ah ok, no problem there, then!